tools/test-harnesses/README.md

```diff
@@ -1 +1,3 @@
 # `@lg-tools/test-harness`
+
+This package provides several fundamental queries that will be utilized by every test harness we ship.
```

Comment:
nit: should be phrased in the present tense: "...that are used by all component specific test harnesses"